src/modules/Builder/components/Configuration/Configuration.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+import { suitableOS, suitableDevices } from "./config";
+
+import Network from "./components/Network/Network";
+import WirelessSupport from "./components/WirelessSupport/WirelessSupport";
+
+export default class Configuration extends Component {
+    constructor(props) {
+        super(props);
+
+        this.serverRefs = {
+            network: React.createRef(),
+            wirelessSupport: React.createRef(),
+        };
+
+        this.buttonStates = {
+            networkHostName: true,
+            wirelessSupportPSK: true,
+        };
+
+        this.builderCallback = () => { };
+
+        this.buttonStateCallback = this.buttonStateCallback.bind(this);
+    }
+
+    executeState(builderCallback) {
+        this.builderCallback = builderCallback.bind(this);
```

Comment:
You don't need create property in `this`, you cat take `builderCallback` from props.